### PR TITLE
[bitnami/metrics-server] Add source repo

### LINKED ISSUE #3551: Chart.yaml for metrics-server should include a link to this repository
**Which chart**:
I'm particularly interested in the metrics-server chart, but _every_ chart would probably benefit from this

**Is your feature request related to a problem? Please describe.**
It's hard to find this github repository from the chart's listing on Helm Hub.

**Describe the solution you'd like**
This `Chart.yaml` file for the metrics-server chart should list https://github.com/bitnami/charts/tree/masterbitnami/metrics-server as either the `home` url or as a `source` url.

### PR DESCRIPTION
**Pull request by miguelaeh**
**Description of the change**
Add source of the repo to the Chart.yaml

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #3551

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by carrodher** [comment]
I think this is something we should apply to all the charts, we should discuss if the appropriate place is `source` or `home` and apply the same change everywhere.

According to the official doc:
```yaml
home: The URL of this project home page (optional)
sources:
  - A list of URLs to source code for this project (optional)
```

cc @juan131 @javsalgar

**Review by dani8art — APPROVED** [review]
LGTM!

File: bitnami/kube-prometheus/Chart.yaml [hunk 1]
```diff
@@ -8,13 +8,14 @@ keywords:
...
   - monitoring
-home: https://github.com/prometheus-operator/kube-prometheus
+home: https://github.com/bitnami/charts/tree/master/bitnami/kube-prometheus
 icon: https://bitnami.com/assets/stacks/prometheus-operator/img/prometheus-operator-stack-220x234.png
 sources:
   - https://github.com/bitnami/bitnami-docker-prometheus-operator
   - https://github.com/bitnami/bitnami-docker-prometheus
   - https://github.com/bitnami/bitnami-docker-alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
+  - https://github.com/prometheus-operator/kube-prometheus
```
Comment (carrodher) [inline bitnami/kube-prometheus/Chart.yaml:18]: This is repeated

File: bitnami/kube-state-metrics/Chart.yaml [hunk 2]
```diff
@@ -7,11 +7,12 @@ keywords:
   - prometheus
   - kube-state-metrics
   - monitoring
-home: https://github.com/kubernetes/kube-state-metrics
+home: https://github.com/bitnami/charts/tree/master/bitnami/kube-state-metrics
 icon: https://bitnami.com/assets/stacks/kube-state-metrics/img/kube-state-metrics-stack-220x234.png
 sources:
   - https://github.com/bitnami/bitnami-docker-kube-state-metrics
   - https://github.com/kubernetes/kube-state-metrics
+  - https://github.com/kubernetes/kube-state-metrics
```
Comment (carrodher) [inline bitnami/kube-state-metrics/Chart.yaml:15]: Same here

**Review by carrodher — APPROVED** [review]
LGTM, thanks!

**Pull request merged**